tools/depends/target/gnutls/Makefile

```diff
@@ -31,7 +30,8 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
 	cd $(PLATFORM); patch -p1 -i ../size-max.patch
-	cd $(PLATFORM); patch -p1 -i ../02-darwin-getentropy.patch
```

Comment:
How does this work on OS X 10.9 (Mavericks), because it doesn't have the `getentropy` function?

Reply:
was told to remove it

Reply:
That was my mistake. Would need to change the min OS X version before this would be possible.